<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register_conteiner_error_vector</class>
 <widget class="QWidget" name="Register_conteiner_error_vector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>50</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>50</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="0,0,0,0,0,0,0,0,0,0,0,0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <widget class="QLabel" name="labelBit7">
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
         <property name="lable_register_brouser" stdset="0">
          <string>set</string>
         </property>
         <property name="lable_error_reg_enabled" stdset="0">
          <string>true</string>
         </property>
         <property name="lable_error_reg_disabled" stdset="0">
          <string>true</string>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QLabel" name="labelBit0">
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>F</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
         <property name="lable_register_brouser" stdset="0">
          <string>set</string>
         </property>
         <property name="lable_error_reg_enabled" stdset="0">
          <string>true</string>
         </property>
         <property name="lable_error_reg_disabled" stdset="0">
          <string>true</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QLabel" name="labelBit2">
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
         <property name="lable_register_brouser" stdset="0">
          <string>set</string>
         </property>
         <property name="lable_error_reg_enabled" stdset="0">
          <string>true</string>
         </property>
         <property name="lable_error_reg_disabled" stdset="0">
          <string>true</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="labelBit4">
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>F</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
         <property name="lable_register_brouser" stdset="0">
          <string>set</string>
         </property>
         <property name="lable_error_reg_enabled" stdset="0">
          <string>true</string>
         </property>
         <property name="lable_error_reg_disabled" stdset="0">
          <string>true</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="labelBit6">
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
         <property name="lable_register_brouser" stdset="0">
          <string>set</string>
         </property>
         <property name="lable_error_reg_enabled" stdset="0">
          <string>true</string>
         </property>
         <property name="lable_error_reg_disabled" stdset="0">
          <string>true</string>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QLabel" name="labelBit1">
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
         <property name="lable_register_brouser" stdset="0">
          <string>set</string>
         </property>
         <property name="lable_error_reg_enabled" stdset="0">
          <string>true</string>
         </property>
         <property name="lable_error_reg_disabled" stdset="0">
          <string>true</string>
         </property>
        </widget>
       </item>
       <item row="1" column="10">
        <widget class="QLabel" name="label_Value">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string> </string>
         </property>
         <property name="lable_register_brouser" stdset="0">
          <string>set</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6" alignment="Qt::AlignHCenter">
        <widget class="QCheckBox" name="checkBoxBit2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="lable_register_brouser" stdset="0">
          <string>set</string>
         </property>
         <property name="QCheckBoxBlue_DisableCheked" stdset="0">
          <string>true</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4" alignment="Qt::AlignHCenter">
        <widget class="QCheckBox" name="checkBoxBit4">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="lable_register_brouser" stdset="0">
          <string>set</string>
         </property>
         <property name="QCheckBoxBlue_DisableCheked" stdset="0">
          <string>true</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="labelBit5">
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
         <property name="lable_register_brouser" stdset="0">
          <string>set</string>
         </property>
         <property name="lable_error_reg_enabled" stdset="0">
          <string>true</string>
         </property>
         <property name="lable_error_reg_disabled" stdset="0">
          <string>true</string>
         </property>
        </widget>
       </item>
       <item row="1" column="12">
        <widget class="QLabel" name="labelDiscription">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="lable_register_brouser" stdset="0">
          <string>set</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelRegNum">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Num</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="lable_register_brouser" stdset="0">
          <string>set</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5" alignment="Qt::AlignHCenter">
        <widget class="QCheckBox" name="checkBoxBit3">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="lable_register_brouser" stdset="0">
          <string>set</string>
         </property>
         <property name="QCheckBoxBlue_DisableCheked" stdset="0">
          <string>true</string>
         </property>
        </widget>
       </item>
       <item row="1" column="9" alignment="Qt::AlignLeft">
        <widget class="QLabel" name="labelValue">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>0x0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="lable_register_brouser" stdset="0">
          <string>set</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3" alignment="Qt::AlignHCenter">
        <widget class="QCheckBox" name="checkBoxBit5">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="lable_register_brouser" stdset="0">
          <string>set</string>
         </property>
         <property name="QCheckBoxBlue_DisableCheked" stdset="0">
          <string>true</string>
         </property>
        </widget>
       </item>
       <item row="1" column="7" alignment="Qt::AlignHCenter">
        <widget class="QCheckBox" name="checkBoxBit1">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="lable_register_brouser" stdset="0">
          <string>set</string>
         </property>
         <property name="QCheckBoxBlue_DisableCheked" stdset="0">
          <string>true</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="labelBit3">
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
         <property name="lable_register_brouser" stdset="0">
          <string>set</string>
         </property>
         <property name="lable_error_reg_enabled" stdset="0">
          <string>true</string>
         </property>
         <property name="lable_error_reg_disabled" stdset="0">
          <string>true</string>
         </property>
        </widget>
       </item>
       <item row="1" column="8" alignment="Qt::AlignHCenter">
        <widget class="QCheckBox" name="checkBoxBit0">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="lable_register_brouser" stdset="0">
          <string>set</string>
         </property>
         <property name="QCheckBoxBlue_DisableCheked" stdset="0">
          <string>true</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" alignment="Qt::AlignHCenter">
        <widget class="QCheckBox" name="checkBoxBit6">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="lable_register_brouser" stdset="0">
          <string>set</string>
         </property>
         <property name="QCheckBoxBlue_DisableCheked" stdset="0">
          <string>true</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" alignment="Qt::AlignHCenter">
        <widget class="QCheckBox" name="checkBoxBit7">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="lable_register_brouser" stdset="0">
          <string>set</string>
         </property>
         <property name="QCheckBoxBlue_DisableCheked" stdset="0">
          <string>true</string>
         </property>
        </widget>
       </item>
       <item row="1" column="11">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QLabel" name="labelInterpritation">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="lable_register_brouser" stdset="0">
            <string>set</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
